ethcore/transaction/tests/mod.rs

```diff
@@ -0,0 +1,76 @@
+#[cfg(test)]
+extern crate heapsize;
+extern crate rustc_hex;
+extern crate ethcore_transaction;
+extern crate ethereum_types;
+extern crate ethkey;
+
+mod tests {
+    use super::*;
+    use ethcore_transaction::*;
+    use ethereum_types::U256;
+
+    #[test]
+    fn heapsize_should_match_std() {
+		assert_eq!(8, std::mem::size_of_val(&5u64));
+		assert_eq!(4, std::mem::size_of_val(&5u32));
+		assert_eq!(1, std::mem::size_of_val(&5u8));
+
+        use heapsize::HeapSizeOf;
```

Comment:
Please clean this to use tab for indentation!

Reply:
Done